utility/src/main/java/org/oppia/util/parser/SvgEncoder.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.util.parser
+
+import com.bumptech.glide.load.EncodeStrategy
+import com.bumptech.glide.load.Options
+import com.bumptech.glide.load.ResourceEncoder
+import com.bumptech.glide.load.engine.Resource
+import com.caverock.androidsvg.SVG
+import java.io.File
+import java.io.FileOutputStream
+
+/** Encodes an SVG internal representation to {@link FileOutputStream}. */
+class SvgEncoder : ResourceEncoder<SVG?> {
+
+  override fun getEncodeStrategy(options: Options): EncodeStrategy {
+    return EncodeStrategy.SOURCE
+  }
+
+  override fun encode(data: Resource<SVG?>, file: File, options: Options): Boolean {
+    return try {
+      val svg: SVG = data.get()
+      val picture = svg.renderToPicture()
+      picture.writeToStream(FileOutputStream(file))
```

Comment:
Where does ``writeToStream`` exist? I don't see it in Android's documentation for ``Picture``.

Reply:
It is Picture's writeToStream method it's undocumented. May be this SO answer can help us https://stackoverflow.com/a/5531302/4500099

Reply:
Ack, thanks. Given it's a private API, we ought to avoid using it since there may be no guarantee for compatibility. Once concern I'd have is that migrating data from one phone to another (e.g. after purchasing a new device) might cause compatibility issues here. Could we instead store the raw SVG file and load that each time?

Reply:
I found that it is deprecated [reference](https://emanual.github.io/Android-docs/reference/android/graphics/Picture.html) . And Document recommends to draw the picture into a Bitmap instead.